source_test.yaml

```diff
@@ -256,15 +256,18 @@
   link: 'https://github.com/RConsortium/r-advisory-database/blob/main/'
   editable: False
 
-- name: 'redhat-stage'
+- name: 'redhat'
   versions_from_repo: False
-  type: 1
-  bucket: 'osv-test-cve-osv-conversion'
-  directory_path: 'redhat-temporary'
+  rest_api_url: 'https://security.access.redhat.com/data/osv/all.json'
+  type: 2
+  directory_path: 'data/osv'
+  detect_cherrypicks: False
   ignore_patterns: ['^(?!RH[BES]{1}A-).*$']
   extension: '.json'
   db_prefix: ['RHBA-', 'RHEA-', 'RHSA-']
-  link: 'https://storage.googleapis.com/osv-test-cve-osv-conversion/'
+  ignore_git: False
+  human_link: 'https://access.redhat.com/errata/{{ BUG_ID }}'
+  link: 'https://security.access.redhat.com/data/osv/'
```

Comment:
This link returns 404 because we don't have a index.html setup. Does it matter?

Reply:
I was just [sanity checking that](https://github.com/google/osv.dev/blob/bb583e85b054e3afec31fc52b5f7ab57cf2cf6f0/gcp/appengine/frontend3/src/templates/vulnerability.html#L35). I don't expect this to be a problem, but it's something to review post-import.